extensions/podman/packages/extension/scripts/podman-download.ts

```
@@ -98,11 +101,17 @@ export class PodmanDownload {
 
     if (this.#platform === 'win32') {
       const tagVersion = podmanJSON.platform.win32.version;
-      const downloadName = podmanJSON.platform.win32.fileName;
+      const downloadNameAmd64 = podmanJSON.platform.win32.arch.x64.fileName;
       this.#artifactsToDownload.push({
         version: tagVersion,
-        downloadName,
-        artifactName: `podman-${this.#podmanVersion}-setup.exe`,
+        downloadName: downloadNameAmd64,
+        artifactName: `podman-installer-windows-amd64.exe`,
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)